### PR TITLE
New package: xnotify-0.9.3

### PR DESCRIPTION
**Pull request by loosefish-scapegrace**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl
  - i686

### CODE REVIEW

File: srcpkgs/xnotify/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+pkgname=xnotify
+version=0.9.3
+revision=1
+build_style=gnu-makefile
+makedepends="imlib2-devel libXft-devel libX11-devel libXinerama-devel"
+short_desc="Read notifications from stdin and pop them up on the screen"
+maintainer="népéta <nepeta@canaglie.net>"
+license="MIT"
+homepage="https://github.com/phillbush/xnotify"
+distfiles="https://github.com/phillbush/xnotify/archive/refs/tags/v0.9.3.tar.gz"
```
Comment (Luciogi) [inline srcpkgs/xnotify/template:11]:
```suggestion
distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
```

Comment (classabbyamp) [reply srcpkgs/xnotify/template:11]: no, don't use homepage in distfiles

File: srcpkgs/xnotify/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+do_build() {
+	[ -e ${FILESDIR}/config.h ] && cp ${FILESDIR}/config.h config.h
+	make INCS="-I${XBPS_CROSS_BASE}/usr/include -I${XBPS_CROSS_BASE}/usr/include/freetype2" \
+		LIBS="-lfontconfig -lXft -lX11 -lXinerama -lImlib2"
+}
+
+do_install() {
+	make PREFIX=/usr DESTDIR=${DESTDIR} install
+	vdoc README.md
+	vlicense LICENSE
```
Comment (sh1r4s3) [inline srcpkgs/xnotify/template:23]:
```suggestion
	vlicense LICENSE
	vman xnotify.1
```

Comment (sh1r4s3) [reply srcpkgs/xnotify/template:23]: Oh, I missed that `Makefile` installs it. Sorry :)

File: srcpkgs/xnotify/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+makedepends="imlib2-devel libXft-devel libX11-devel libXinerama-devel"
+short_desc="Read notifications from stdin and pop them up on the screen"
+maintainer="népéta <nepeta@canaglie.net>"
+license="MIT"
+homepage="https://github.com/phillbush/xnotify"
+distfiles="https://github.com/phillbush/xnotify/archive/refs/tags/v0.9.3.tar.gz"
+checksum=9fa7505ad16c8d5e2305a56d0d88ca3a7e324816b4df7845b920db088482eb24
+
+do_build() {
+	[ -e ${FILESDIR}/config.h ] && cp ${FILESDIR}/config.h config.h
```
Comment (ahesford) [inline srcpkgs/xnotify/template:15]: I don't see anywhere where this is respected by the build process.

Comment (loosefish-scapegrace) [reply srcpkgs/xnotify/template:15]: isn't it? the xnotify makefile brings in `config.h`. (the upstream xnotify no longer has that file, but the latest release still does)

Comment (ahesford) [reply srcpkgs/xnotify/template:15]: That's even more of an argument against adding this capability. Some future release will drop it entirely and we'll be left with vestigial logic that only existed to support custom builds in the first place.

**Comment by loosefish-scapegrace** [comment]
"Andrew J. Hesford" ***@***.***> wrote:
> @ahesford commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,24 @@
> +# Template file for 'xnotify'
> +pkgname=xnotify
> +version=0.9.3
> +revision=1
> +build_style=gnu-makefile
> +makedepends="imlib2-devel libXft-devel libX11-devel libXinerama-devel"
> +short_desc="Read notifications from stdin and pop them up on the screen"
> +maintainer="népéta ***@***.***>"
> +license="MIT"
> +homepage="https://github.com/phillbush/xnotify"
> +distfiles="https://github.com/phillbush/xnotify/archive/refs/tags/v0.9.3.tar.gz"
> +checksum=9fa7505ad16c8d5e2305a56d0d88ca3a7e324816b4df7845b920db088482eb24
> +
> +do_build() {
> +	[ -e ${FILESDIR}/config.h ] && cp ${FILESDIR}/config.h config.h
> 
> That's even more of an argument against adding this capability. Some future release will drop it entirely and we'll be left with vestigial logic that only existed to support custom builds in the first place.

That's fair, I think. `xnotify` supports Xresources anyway, so I
think it's not as useful as for something like `dmenu`...

File: srcpkgs/xnotify/template (hunk 3 above)
Comment (loosefish-scapegrace) [reply srcpkgs/xnotify/template:15]: > That's even more of an argument against adding this capability. Some future release will drop it entirely and we'll be left with vestigial logic that only existed to support custom builds in the first place.

That's fair, I think. `xnotify` supports Xresources anyway, so I
think it's not as useful as for something like `dmenu`...

**Comment by loosefish-scapegrace** [comment]
modified the template to use `make_build_args` and remove the `config.h` line

**Pull request closed without merging**